charts/heimdall/templates/podmonitor.yaml

```
@@ -0,0 +1,46 @@
+{{- $podmonitorsCRD := ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) }}
+{{- $metrics := default dict .Values.metrics }}
+{{- if and
+  (ne $metrics.enabled false)
+  (contains "prometheus" ( .Values.env.OTEL_METRICS_EXPORTER | default ""))
```

```suggestion
  (eq "prometheus" ( .Values.env.OTEL_METRICS_EXPORTER | default ""))
```

Prefer using `eq` for an exact equality check rather than `contains`